src/components/UI/MultiChoiceMenu/MultiChoiceMenu.js

```diff
@@ -7,7 +7,7 @@ import {MultiChoiceErrorMessage} from '../MultiChoiceErrorMessage/MultiChoiceErr
 import {MultiChoiceItem} from '../MultiChoiceItem/MultiChoiceItem';
 import styles from './MultiChoiceMenu.module.scss';
 
-export const MultiChoiceMenu = ({title, description, choices, error, footer}) => {
+export const MultiChoiceMenu = ({title, description, choices, error, footer, toCenter}) => {
```

Comment:
Why we need `toCenter`?

Reply:
We use it on the `<Liquidity />` component, we need it to be centralized, and on `<LoginModal />`, we don't need it to be centralized

Reply:
Isn't both centralised?

Reply:
No, we talk about it yesterday, and we understand that we need to remove the class: `center` from the `<LoginModal />` but in the `<Liquidity />` component we need this class